README.md

```diff
@@ -49,7 +49,7 @@ Licensed under the [MIT License](LICENSE.md).
 [@axic]: https://github.com/axic
 [@chfast]: https://github.com/chfast
 [documentation]: https://ethereum.github.io/evmc
-[eWASM]: https://github.com/ewasm/design#ethereum-flavored-webassembly-ewasm-design
+[Ewasm]: https://github.com/ewasm/design#ethereum-flavored-webassembly-ewasm-design
```

Comment:
This anchor changed, I think now it's https://github.com/ewasm/design#what-is-ethereum-flavored-webassembly-ewasm